docs/source/community/rfc/rfc-6.rst

```diff
@@ -0,0 +1,416 @@
+correction values is really easy and efficient, so triangulation-based transformations
+are not meant as replacing them, but more about it being a complement, that is
+sometimes necessary to be able to replicate the results of a officially vetted
+transformation to a millimetric or better precision (speaking here about reproducibility
+of numeric results, rather than the physical accuracy of the transformation that
+might rather be centimetric). It is always possible to approach the result of
+the triangulation with a grid, but that may require to adopt a small grid step,
+and thus generate a grid that can be much larger than the original triangulation.
+
+Details
+-------------------------------------------------------------------------------
+
+Transformation
+++++++++++++++
+
+A new transformation method, ``tinshift``, is added. It takes one mandatory
+argument, ``file``, that points to a JSON file, which contains the triangulation
+and associated metadata. Input and output coordinates can be in geographic
+or in projected coordinates, depending on the metadata of the JSON file
+describing the input and output CRS of the transformation. Depending on the content
+of the JSON file, horizontal, vertical or both components of the coordinates may
+be transformed.
+
+So, in its most simple invocation, the transformation is used like:
+
+::
+
+    $ echo  21.8214426073 59.8602198556 0 | cct +proj=tinshift +file=./triangulation_kkj.json
+
+    21.8181756139   59.8602955636        0.0000           inf
+
+The transformation is invertible, with the same computational complexity than
+the forward transformation.
+
+The KKJ to ETRS89 transformation has the particularity that the vertices are
+defined in projected coordinates, rather than geographic coordinates. The
+JSON file describes in that situation four CRS:
+
+- the CRS of the input coordinates to provide to ``tinshift``: EPSG:4123 ("KKJ"), geographic CRS
+- the CRS of the source coordinates in the triangulation: EPSG:2393 ("KKJ / Finland Uniform Coordinate System"), projected CRS
+- the CRS of the target coordinates in the triangulation: EPSG:3067 ("ETRS89 / TM35FIN(E,N)"), projected CRS
+- the CRS of the output coordinates returned by``tinshift``: EPSG:4258 ("ETRS89"), geographic CRS
+
+From a user point of view, the fact that projected coordinates are used internally is
+mostly an implementation detail. For users seeking for maximum performance, and that
+have to transform coordinates in the source CRS of the triangulation (resp. expect output coordinates
+in the target CRS of the triangulation), they can provide a ``+skip_input_conv`` parameter
+(resp. ``+skip_output_conv``) to avoid the input (resp. output) conversion to be done.
+
+Example, equivalent to the previous one, using the projected coordinates:
+
+::
+
+    $ echo  3210000 6650000 0 |  cct +proj=tinshift +file=./triangulation_kkj.json +skip_input_conv +skip_output_conv
+
+    209948.5283   6647207.3168        0.0000           inf
```

Comment:
Add a time value to avoid the `inf` output, which might cause confusion for readers